showcase/app/components/shw/logo/design-system.gts

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+
+export default class ShwLogoDesignSystemComponent extends Component {
+  <template>
+    <svg
+      class="shw-logo-hds__logo"
+      width="42"
+      height="42"
+      viewBox="0 0 42 42"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      aria-hidden="true"
+    >
+      <defs>
+        <linearGradient
```

Comment:
Note: without changing the linting rules, this file would always error saying that linearGradient isn't defined.